### ISSUE TITLE
Add Exercism public profile

### ISSUE THREAD

**Issue by RianNegreiros** [rahuldkjain/github-profile-readme-generator#761]
Add Exercism public profile in Social, like HackerRank, LeetCode, CodeChef etc…

**Comment by nik132-eng** [comment]
please assign me this issue so I can work on this issue.

**Comment by Tanush-J** [comment]
For contributing am I suppose to create a branch in this project itself or in my fork?

**Comment by bhavy03** [comment]
I would like to work on this issue

**Comment by sankettank66** [comment]
Could you assign this issue to me?

**Comment by azizarizaldi** [comment]
please sir assign me this issue so I can work on this issue. 😁 (bcs I'm a beginner)

**Comment by RianNegreiros** [comment]
Unfortunately, only contributors can assign someone to an issue, and the last time this happened was this in [issue](https://github.com/rahuldkjain/github-profile-readme-generator/issues/598) from over a year ago.

**Comment by shivamshahi07** [comment]
I think socials like hacker rank, leet-code, and code forces are already available on the read-me generator

**Comment by RianNegreiros** [comment]
@shivamshahi07 Yes, my suggestion is to also add the [exercism](https://exercism.org/) which is a platform like HackerRank, LeetCode, CodeChef

**Comment by shivamshahi07** [comment]
@RianNegreiros OK, my bad I was actually not familiar with exercism . 
thanks

**Comment by RianNegreiros** [comment]
@shivamshahi07 No problem
